innopoints/core/helpers.py

```diff
@@ -23,6 +23,9 @@ def abort(http_code: int, message=None):
 
 def csrf_protect():
     """Validates the CSRF token for modifying requests (POST, PATCH, DELETE)."""
+    if current_app.config['SKIP_CSRF_CHECK']:
```

Comment:
I suggest either adding `SKIP_CSRF_CHECK = False` to the common.py config so that it would exist by default, changing this line to make the key optional, or simply using `FLASK_ENV` instead.